### PR TITLE
Cbapi 4828 rtd guide

### PR DESCRIPTION
**Pull request by heiderje-vmware**
## Pull request checklist

Please check if your PR fulfills the following requirements:

- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Tests have been added that prove the fix is effective or that the feature works.
- [x] New and existing tests pass locally with the changes.
- [x] Code follows the style guidelines of this project (PEP8, clean code, linter).
- [x] A self-review of the code has been done.

## What is the ticket or issue number?
[CBAPI_4828](https://jira.carbonblack.local/browse/CBAPI-4828)

## Pull Request Description
updated RTD for group alerts

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## How Has This Been Tested?
local build of RTD

### CODE REVIEW

**Comment by heiderje-vmware** [comment]
Would like any general feedback but also would like someone to help with verbage surrounding grouped alert facets. I'm not sure what to say about them.
Also please take note of the method names surrounding conversion from an AlertSearchQuery <> GroupedAlertSearchQuery. it feels odd to me that set_group_by() converts the first and get_alert_search_query() converts the latter. coming back to it after stepping away from it for some time it almost feels as if set_group_by() on AlertSearchQuery should be renamed to get_grouped_alert_search_query()  to seem less disjointed.
thoughts?

File: docs/alerts.rst [hunk 1]
```diff
@@ -152,6 +152,49 @@ You can also read from a csv file by using values that match the profile names i
     >>> for alert in alerts:
     ...     print(alert.id, alert.device_os, alert.device_name, alert.category)
 
+Grouping Alerts
+---------------
+
+The examples below show illustrate how to create and manipulate grouped alert objects. A Grouped Alert is a collections of alerts that share a common threat id. When grouping alerts by a threat id it allows greater context and insight surrounding the pervasiveness of a threat.
```
Comment (kebringer-cb) [inline docs/alerts.rst:158]: type - "...below show illustrate how...", just one of show or illustrate

File: docs/alerts.rst [hunk 2]
```diff
@@ -152,6 +152,49 @@ You can also read from a csv file by using values that match the profile names i
...
+It may be necessary to retrieve all of the alerts from a threat id grouping for further inspection, it is possible to directly retrieve the associated alert search query from a given grouped alert
+
+    >>> alert_search_query = group_alert.get_alert_search_query()
+    >>> alerts = alert_search_query.all()
+
+It is also possible to create grouped facets from the group alert search query
+
+    >>> grouped_alert_facets = group_alert_search_query.facets(["type", "THREAT_ID"], 0, True)
+
+Suppose instead of grouped alerts, you had been working with alerts and wanted to crossover to grouped alerts. Instead of building a new group alert query from scratch you can transform an alert search query into a grouped alert search query or vice versa!
```
Comment (kebringer-cb) [inline docs/alerts.rst:188]: This shows in bold.  I think it just needs a empty line between 188 and the example at 189.

**Review by kebringer-cb — APPROVED** [review]
Couple of typos to fix, then good to merge.

**Review by abowersox-cb — APPROVED** [review]
Other than what Kylie has marked, I don't see anything wrong here.

**Review by avanbrunt-cb — APPROVED** [review]

**Pull request merged**